tmail-backend/blob/blobid-list/src/main/java/com/linagora/tmail/blob/blobid_list/cassandra/CassandraSingleSaveBlobStoreDAO.java

```diff
@@ -0,0 +1,37 @@
+package com.linagora.tmail.blob.blobid_list.cassandra;
+
+import com.linagora.tmail.blob.blobid_list.SingleSaveBlobStoreDAO;
+import org.apache.james.backends.cassandra.init.configuration.CassandraConfiguration;
+import org.apache.james.blob.api.BlobId;
+import org.apache.james.blob.api.BucketName;
+import org.apache.james.blob.cassandra.CassandraBlobStoreDAO;
+import org.apache.james.blob.cassandra.CassandraBucketDAO;
+import org.apache.james.blob.cassandra.CassandraDefaultBucketDAO;
+import reactor.core.publisher.Mono;
+
+import javax.inject.Inject;
+
+public class CassandraSingleSaveBlobStoreDAO extends CassandraBlobStoreDAO {
```

Comment:
Why do we need this class at all?

Reply:
We need one element of `BlobStoreDAO` interface, right?
If We extend `CassandraBlobStoreDAO`,  we will only need to override the `save` and `delete` method (not all). 
I saw you wrote: 
```
// Write a BlobStoreDAO that decorates another BlobStoreDAO 
class SingleSaveBlobStoreDAO(blobStoreDAO, cassandraCreatedBlobIdStore) {
   ...
}
```
I can't find anyway, that how can I create a dependency (interface implement) with only SingleSaveBlobStoreDAO.